src/main/java/com/formulasearchengine/mathmltools/mml/CMMLInfo.java

```diff
@@ -771,6 +741,18 @@ public final CMMLInfo toStrictCmmlCont() {
         return this;
     }
 
+
+    public final String toString(boolean useCache) {
```

Comment:
can you add some documentation? I get the feeling that this might cause problems...